admin/acceptance.py

```diff
@@ -163,12 +163,14 @@ class VagrantRunner(object):
     # rather than assuming it is available.
     # https://clusterhq.atlassian.net/browse/FLOC-1163
 
-    NODE_ADDRESSES = ["172.16.255.240", "172.16.255.241"]
+    NODE_ADDRESSES = ["172.16.255.250", "172.16.255.251"]
```

Comment:
So does this mean that you can't just swap the IP addresses in the tutorial like it says if there is a conflict with your network?